corporate/lib/stripe.py

```diff
@@ -737,11 +749,16 @@ def get_data_for_stripe_payment_intent(
                 "Support requests do not set any stripe billing information.",
             )
 
-        amount = price_per_license * licenses
-
         plan_name = CustomerPlan.name_from_tier(plan_tier)
-        description = f"Upgrade to {plan_name}, ${price_per_license/100} x {licenses}"
-        if customer.flat_discounted_months > 0:
+        if price_per_license is not None:
+            amount = price_per_license * licenses
+            description = f"Upgrade to {plan_name}, ${price_per_license/100} x {licenses}"
+        else:
+            assert fixed_price is not None
+            amount = get_amount_due_fixed_price_plan(fixed_price, billing_schedule)
+            description = plan_name
```

Comment:
Should we have an `assert fixed_price is None or price_per_license is None` above this block?

Reply:
Yeah, added that assert statement.